dll/win32/uxtheme/system.c

```diff
@@ -1153,32 +1162,65 @@ HRESULT WINAPI CloseThemeFile(HTHEMEFILE hThemeFile)
  *
  * PARAMS
  *     hThemeFile           Handle to theme file
- *     unknown              See notes
+ *     Flags                Unknown flag bits
  *     hWnd                 Window requesting the theme change
  *
  * RETURNS
  *     Success: S_OK
  *     Failure: HRESULT error-code
- *
- * NOTES
- * I'm not sure what the second parameter is (the datatype is likely wrong), other then this:
- * Under XP if I pass
- * char b[] = "";
- *   the theme is applied with the screen redrawing really badly (flickers)
- * char b[] = "\0"; where \0 can be one or more of any character, makes no difference
- *   the theme is applied smoothly (screen does not flicker)
- * char *b = "\0" or NULL; where \0 can be zero or more of any character, makes no difference
- *   the function fails returning invalid parameter... very strange
  */
+#ifdef __REACTOS__
+HRESULT WINAPI ApplyTheme(HTHEMEFILE hThemeFile, UINT Flags, HWND hWnd)
+{
+    HRESULT hr;
+    TRACE("(%p,%#x,%p)\n", hThemeFile, Flags, hWnd);
+#else
 HRESULT WINAPI ApplyTheme(HTHEMEFILE hThemeFile, char *unknown, HWND hWnd)
 {
     HRESULT hr;
     TRACE("(%p,%s,%p)\n", hThemeFile, unknown, hWnd);
+#endif
```

Comment:
Let's not break code editors block folding with the unbalanced opened bracket:
```suggestion
#ifdef __REACTOS__
HRESULT WINAPI ApplyTheme(HTHEMEFILE hThemeFile, UINT Flags, HWND hWnd)
#else
HRESULT WINAPI ApplyTheme(HTHEMEFILE hThemeFile, char *unknown, HWND hWnd)
#endif
{
    HRESULT hr;
#ifdef __REACTOS__
    TRACE("(%p,%#x,%p)\n", hThemeFile, Flags, hWnd);
#else
    TRACE("(%p,%s,%p)\n", hThemeFile, unknown, hWnd);
#endif
```